packages/uploadthing/src/internal/express.ts

```diff
@@ -0,0 +1,88 @@
+import { Router as ExpressRouter } from "express";
+
+import { getStatusCodeFromError, UploadThingError } from "@uploadthing/shared";
+
+import { UPLOADTHING_VERSION } from "../constants";
+import { defaultErrorFormatter } from "./error-formatter";
+import type { RouterWithConfig } from "./handler";
+import { buildPermissionsInfoHandler, buildRequestHandler } from "./handler";
+import { getPostBody } from "./node-http/getBody";
+import type { FileRouter, inferErrorShape } from "./types";
+
+export const createExpressRouter = <TRouter extends FileRouter>(
```

Comment:
the name `createExpressRouter` doesn't quite make sense to me? should it not be `createExpressHandler` or something? the next.js ones are named like that.

even if express uses the `Router` term, i think it would confuse some what the difference of a `FileRouter` and an `ExpressRouter` would be

Reply:
createUTExpressHandler?